cluster/environments/azure-common-infra/keyvault.tf

```diff
@@ -1,23 +1,26 @@
 data "azurerm_client_config" "current" {}
 
-module "keyvault" "keyvault" {
-  source = "github.com/Microsoft/bedrock/cluster/azure/keyvault"
+module "keyvault" {
+  #source = "github.com/Microsoft/bedrock/cluster/azure/keyvault"
+  source = "../../azure/keyvault"
```

Comment:
Doesn't this mean you need a clone the entire repo and can't copy out the environments you want? Does source no longer allow github urls?

Reply:
The source url bit is only to pass the test - pre-review.  We revert and force merge after review.

This is due to the fact that changes are being made to some portion of the source which differs from the remote one.

Only method found for this.

Reply:
👍